### DISCUSSION TITLE
(ArtifactUpload>DataPipeline>XXX) Update tag in Jenkins configuration manually

### DISCUSSION THREAD

**Discussion by daveanvit** [Sunbird-Ed/Community#122] [Documentation]
For uploading artifact for datapipelines configure release tag for each job manually in place of **public_repo_branch** otherwise it will fail as it is taking release tag for **jenkins.yaml** file as **release-4.10.1_RC1** and release tag is not present in https://github.com/project-sunbird/sunbird-data-pipeline.git

Use tag [release-4.4.0_RC2](https://github.com/Sunbird-Ed/Community/discussions/82#discussioncomment-3453605) / If you face issue using **release-4.4.0_RC2** tag apply specific tag for that job.

Documenting for further installation

**Comment by beepdot** [comment]
Thank you for the information.

**Comment by sharathkashyap** [comment]
@daveanvit Thank you for your contribution. We will be locking this thread due to inactivity.
